src/main/drivers/vtx_common.c

```diff
@@ -163,4 +221,23 @@ bool vtxCommonGetDeviceCapability(vtxDeviceCapability_t *pDeviceCapability)
     memcpy(pDeviceCapability, &vtxDevice->capability, sizeof(vtxDeviceCapability_t));
     return true;
 }
+
+vtxSettingsConfig_t vtxCommonGetSettings(void)
+{
+    return (vtxSettingsConfig_t) {
+        .band = vtxSettingsConfig()->band,
+        .channel = vtxSettingsConfig()->channel,
+        .freq = vtxSettingsConfig()->freq,
+        .power = vtxSettingsConfig()->power,
+    };
+}
+
+void vtxCommonUpdateSettings(vtxSettingsConfig_t config)
```

Comment:
Don't pass structures on the stack, so use:
```
void vtxCommonUpdateSettings(vtxSettingsConfig_t *config)
{
    vtxSettingsConfigMutable()->band = config->band;
... etc
```

Also this function should move to `io/vtx_common.c`